src/routes/user/register/1.svelte

```diff
@@ -0,0 +1,384 @@
+		const zxcvbnJaPackage = await import('@zxcvbn-ts/language-ja');
+		const zxcvbnNlBePackage = await import('@zxcvbn-ts/language-nl-be');
+		const zxcvbnPlPackage = await import('@zxcvbn-ts/language-pl');
+		const zxcvbnPtBrPackage = await import('@zxcvbn-ts/language-pt-br');
+
+		const options: OptionsType = {
+			translations: zxcvbnEnPackage.translations,
+			graphs: zxcvbnCommonPackage.adjacencyGraphs,
+			dictionary: {
+				...zxcvbnCommonPackage.dictionary,
+				...zxcvbnCsPackage.dictionary,
+				...zxcvbnDePackage.dictionary,
+				...zxcvbnEsEsPackage.dictionary,
+				...zxcvbnFiPackage.dictionary,
+				...zxcvbnFrPackage.dictionary,
+				...zxcvbnIdPackage.dictionary,
+				...zxcvbnItPackage.dictionary,
+				...zxcvbnJaPackage.dictionary,
+				...zxcvbnNlBePackage.dictionary,
+				...zxcvbnPlPackage.dictionary,
+				...zxcvbnPtBrPackage.dictionary,
+				...zxcvbnEnPackage.dictionary,
+
+				// Check inputs
+				userInputs: [password?.value ?? '', email?.value ?? '', confirm_password?.value ?? '']
+			},
+			useLevenshteinDistance: true
+		};
+
+		zxcvbnOptions.setOptions(options);
+	};
+
+	$effect.pre(() => {
+		// register_languages();
+	});
+
+	// bind:value needs to be defined first to bind
+	// otherwise shows: typeerror $.get(...) is undefined
+	let email = $state<FormFieldState>({ value: "", error: new Array<string>() }),
+		password = $state<FormFieldState>({ value: "", error: new Array<string>() }),
+		confirm_password = $state<FormFieldState>({ value: "", error: new Array<string>() });
+
+	$effect(() => {
+		// update only if state exists on combined_state
+		if (combined_state.email) {
+			email = {
+				value: combined_state?.email ?? "",
+				error: new Array<string>()
+			};
+		};
+		if (combined_state.password) {
+			password = {
+				value: combined_state?.password ?? "",
+				error: new Array<string>()
+			};
+		};
+		if (combined_state.confirm_password) {
+			confirm_password = {
+				value: combined_state?.confirm_password ?? "",
+				error: new Array<string>()
+			};
+		};
+	});
+
+	let password_strength = $state(0);
+	let form_is_submitable = $state(false);
+
+	function check_if_form_is_submitable() {
+		form_is_submitable = [email, password, confirm_password].every((field) => {
+			return field?.value && _.isEmpty(field.error);
+		});
+	}
+
+	const password_error_mapping: { [key: string]: string } = {
+		atleast_8: 'minimum 8 characters',
+		missing_one_special_character: 'minimum 1 special character',
+		missing_one_number: 'minimum 1 number',
+		missing_one_upper_or_lowercase: 'minimum 1 lower-case or upper-case character'
+	};
+
+	// Bindings
+
+	function handle_submit() {
+		if (form_is_submitable) {
+			on_submit({
+				email: email?.value,
+				password: password?.value
+			});
+		}
+		// Show some errors here
+	}
+
+	const handle_email_input = async (event: Event) => {
+			handle_input({
+				event: event,
+				schema: z.string().email('Please enter a valid email address'),
+				error_field: email!
```

Comment:
You can explicitly check for 

```js
if (!email) return;
```

Reply:
Resolved on https://github.com/baseplate-admin/coreproject-v4-ui/pull/35/commits/947d20c094c184140ab7196d61a63edaac301bab